homeassistant/components/device_tracker/tado.py

```diff
@@ -0,0 +1,109 @@
+"""
+homeassistant.components.device_tracker.tado.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Device tracker platform that supports presence detection.
+The detection is based on geofencing enabled devices used with Tado.
+Tado is a Smart Thermostat device.
+"""
```

Comment:
It would be appreciated if your use the same style for the header as the other files do.